server/services/employers.ts

```diff
@@ -168,6 +167,19 @@ export const getSiteByID = async (id: number): Promise<EmployerSite> => {
     throw new Error(`No site found with id ${id}`);
   }
 
+  const currentAllocation = await dbClient.runRawQuery(
```

Comment:
Make returned shape from this query clearer with a type annotation.
```suggestion
  const allocationResponse: [{ allocation: number }?] = await dbClient.runRawQuery(
```